examples/zaps-nwc.js

```diff
@@ -0,0 +1,40 @@
+require('dotenv').config()
+import { LightningAddress, NostrProvider } from "alby-tools";
+import { webln } from "alby-js-sdk";
+import 'websocket-polyfill';
+import * as crypto from "crypto";
+globalThis.crypto = crypto;
+
+(async () => {
+
+  const nostrWalletConnectUrl = process.env.NWC_URL;
+  if (!nostrWalletConnectUrl) {
+    throw new Error("Please add NWC_URL to .env. You can get one at nwc.getalby.com")
+  }
+  const nostrWeblnProvider = new webln.NostrWebLNProvider({ nostrWalletConnectUrl }) // loadNWCUrl() depending on your app. See alby-js-sdk readme
+  const nostrProvider = {
+    getPublicKey: () => Promise.resolve(nostrWeblnProvider.publicKey),
+    signEvent: (event) => Promise.resolve({...event, sig: nostrWeblnProvider.signEvent(event)})
```

Comment:
is this even needed? or is this only needed because the `NostrWebLNProvider` does not do it async?

Reply:
@bumi the response type is also different (the whole event vs only the signature)

Reply:
should we change signEvent then?

Reply:
@bumi sounds good to me! I can make a separate PR for this

Reply:
Fix in https://github.com/getAlby/alby-js-sdk/pull/24

Reply:
actually as I think about it...this is not a good idea, is it? 
one should not sign the zap event with the NWC key. 
That should always be done with the user's key. 

probably still good to implement the same API, but I am not sure if we should do an example like this. 
🤔

Reply:
@bumi good catch 😱

Reply:
I made a custom provider to the examples using nostr-tools, this time with the user's private key. Not sure there is a simpler way to do it right now. alby-js-sdk uses nostr-tools already, maybe we could add some utility function in alby-js-sdk that generates a nostr/NIP-07 provider, and you just give it a private key.

Reply:
why would you use the alby-sdk for that and not nostr-tools?

Reply:
Maybe we could create a PR in nostr-tools for a util function that can make a provider? It would be nice to also expose the NIP07 type there.